src/lib/y2storage/clients/inst_prepdisk.rb

```diff
@@ -83,12 +89,38 @@ def mount_in_target(path, device, options)
         log.info "Cmd: mount #{options} #{device} #{target_path}"
 
         if !SCR.Execute(path(".target.mount"), [device, target_path], options)
-          raise ".target.mount failed"
+          # TRANSLATORS: %s is the path of a system mount like "/dev", "/proc", "/sys"
+          Yast::Report.Warning(_("Could not mount %s") % path )
```

Comment:
Rubocop is going to complain about the extra space after `path`.

Reply:
OK; fixed.